edge-apps/tfl-bus-status/README.md

```diff
@@ -0,0 +1,64 @@
+# TFL Bus Information Edge App
+
+This guide will walk you through the steps to create an TFL Bus Status Edge App in Screenly.
+
+
+## Prerequisites
+
+* Install Screenly CLI - Please follow the guide [here](https://github.com/Screenly/cli)
+
+## Steps
+
+Step 1. **Log in to the Screenly account via CLI**
+
+Follow the on-screen instructions to log in to your Screenly account.
+
+   `$ screenly login`
```

Comment:
Either do four indentations (means code), use the tick marks (single command) or use a code block. This is redundant.